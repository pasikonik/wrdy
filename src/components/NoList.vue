<template>
  <main class="d-flex flex-column align-center">
    <h2 class="text-h2 ma-15">You don't have any list</h2>

    <h3 clas="text-h3">Create new one to add some words</h3>
  </main>
</template>
