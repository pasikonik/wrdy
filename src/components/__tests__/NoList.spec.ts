import { render } from '@testing-library/vue'
import NoList from '../NoList.vue'
import { test, expect } from 'vitest'

test('Check displaying text', () => {
  const { getByText } = render(NoList)

  getByText("You don't have any list")
  getByText('Create new one to add some words')
})

test('Test snapshot', () => {
  const result = render(NoList)
  expect(result).toMatchSnapshot()
})
