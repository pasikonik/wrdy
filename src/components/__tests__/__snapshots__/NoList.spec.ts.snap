// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Test snapshot 1`] = `
{
  "baseElement": <body>
    <div>
      <main
        class="d-flex flex-column align-center"
      >
        <h2
          class="text-h2 ma-15"
        >
          You don't have any list
        </h2>
        <h3
          clas="text-h3"
        >
          Create new one to add some words
        </h3>
      </main>
    </div>
    <div>
      <main
        class="d-flex flex-column align-center"
      >
        <h2
          class="text-h2 ma-15"
        >
          You don't have any list
        </h2>
        <h3
          clas="text-h3"
        >
          Create new one to add some words
        </h3>
      </main>
    </div>
  </body>,
  "container": <div>
    <main
      class="d-flex flex-column align-center"
    >
      <h2
        class="text-h2 ma-15"
      >
        You don't have any list
      </h2>
      <h3
        clas="text-h3"
      >
        Create new one to add some words
      </h3>
    </main>
  </div>,
  "debug": [Function],
  "emitted": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "html": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
